src/sql/parts/jobManagement/views/jobsView.component.ts

```diff
@@ -133,7 +133,8 @@ export class JobsViewComponent implements AfterContentChecked {
 			syncColumnCellResize: true,
 			enableColumnReorder: false,
 			rowHeight: 45,
-			enableCellNavigation: true
+			enableCellNavigation: true,
+			forceFitColumns: true
```

Comment:
will this make some columns more narrow than the minWidth?

Reply:
No, it won't.